src/client/automation/playback/type/type-text.js

```diff
@@ -109,6 +109,10 @@ function _excludeInvisibleSymbolsFromSelection (selection) {
     return selection;
 }
 
+function simulateBeforeInput (element, text) {
```

Comment:
Can we get rid of `simulateBeforeInput` and call `eventSimulator.beforeInput` directly?

Reply:
we can